bindings_ffi/src/mls.rs

```diff
@@ -330,6 +330,15 @@ impl FfiConversations {
                 .add_members(&self.inner_client, account_addresses)
                 .await?;
         }
+
+        if let Some(ref name) = opts.group_name {
+            if !name.is_empty() {
+                convo
+                    .update_group_name(&self.inner_client, name.to_string())
```

Comment:
Ya I was curious about that. Felt like it would have multiple intents getting committed just on create especially as the metadata on create grows. I'll dig deeper.